pages/resources/discovery.mdx

```diff
@@ -0,0 +1,443 @@
+| banner                     | ?string                                                       | The guild's [banner hash](/reference#cdn-formatting)                                             |
+| home_header                | ?string                                                       | The guild's [home header hash](/reference#cdn-formatting), also used in server guide             |
+| splash                     | ?string                                                       | The guild's [splash hash](/reference#cdn-formatting)                                             |
+| discovery_splash           | ?string                                                       | The guild's [discovery splash hash](/reference#cdn-formatting)                                   |
+| features                   | array[string]                                                 | Enabled [guild features](/resources/guild#guild-features)                                        |
+| vanity_url_code            | ?string                                                       | The guild's vanity invite code                                                                   |
+| preferred_locale           | string                                                        | The preferred locale of the guild; used in discovery and notices from Discord (default "en-US")  |
+| premium_subscription_count | integer                                                       | The number of premium subscriptions (boosts) the guild currently has                             |
+| approximate_member_count   | integer                                                       | Approximate number of members in the guild                                                       |
+| approximate_presence_count | integer                                                       | Approximate number of online members in the guild                                                |
+| emojis? ^1^                | array[[emoji](/resources/emoji#emoji-object) object]          | Custom guild emojis; limited to 30 entries                                                       |
+| emoji_count? ^1^           | integer                                                       | Total number of custom guild emojis                                                              |
+| stickers? ^1^              | array[[sticker](/resources/sticker#sticker-object) object]    | Custom guild stickers; limited to 30 entries                                                     |
+| sticker_count? ^1^         | integer                                                       | Total number of custom guild stickers                                                            |
+| auto_removed               | boolean                                                       | Whether the guild has automatically been removed from discovery for not hitting required targets |
```

Comment:
Why is this in a discoverable guild object? Shouldn’t guilds with this be removed from Algolia automatically?

Reply:
I didn't design the API 😭
I don't think it's removed that quickly, and the client filters out auto-removed guilds when searching: `query=test&filters=auto_removed%3Afalse+AND+approximate_presence_count%3E+0+AND+approximate_member_count%3E200&optionalFilters=%5B%22preferred_locale%3A+en-US%22%5D&length=12&offset=0&restrictSearchableAttributes=%5B%22name%22%2C%22description%22%2C%22keywords%22%2C%22categories.name%22%2C%22categories.name_localizations.en-US%22%2C%22primary_category.name%22%2C%22primary_category.name_localizations.en-US%22%2C%22vanity_url_code%22%5D`. Also, there's a grace period before the guild is removed.